app/src/main/java/mozilla/lockbox/view/CreateItemFragment.kt

```diff
@@ -56,4 +57,8 @@ class CreateItemFragment : ItemMutationFragment(), CreateItemView {
         view.inputUsername.hint = getString(R.string.create_username_hint_text)
         view.inputPassword.hint = getString(R.string.hint_password)
     }
-}
+
+    override fun getToastAction(strId: Int?): RouteAction {
+        return RouteAction.ShowToastNotification(strId = strId, viewGroup = this.view as ViewGroup)
```

Comment:
Removing the `viewGroup` requirement on the `Action` would make this a lot simpler.